app/tests/unit/lib/theme/widgets/ContextualHelp.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from "@testing-library/react";
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+import ContextualHelp from "lib/theme/widgets/ContextualHelp";
+
+describe("ContextualHelp component", () => {
+  const titleMock = "Tooltip Title";
+  const labelMock = "Field Label";
+
+  test("renders the tooltip with correct title and label and matches the snapshot", () => {
+    const componentUnderTest = render(
+      <ContextualHelp
+        title={titleMock}
+        icon={faExclamationCircle}
+        label={labelMock}
+      />
+    );
+    const tooltipElement = screen.getByRole("tooltip");
+    const tooltipIcon = screen.getByRole("img", {
+      hidden: true,
```

Comment:
Why are they hidden?

Reply:
The icon has an `img` role but when we want to get it using `getByRole` it doesn't find an accessible element with that role, but if we pass `hidden: true` since the SVG has the `aria-hidden=true` by default, we can select that element for testing purposes.
Also why `aria-hidden=true` on `img` role?: https://www.w3.org/TR/wai-aria-1.2/#tree_exclusion